src/couch_replicator_scheduler.erl

```diff
@@ -226,16 +233,21 @@ oldest_job_first(#job{} = A, #job{} = B) ->
 
 not_recently_crashed(#job{} = Job) ->
     case crash_history(Job) of
-        [] ->
-            true;
-        [{crashed, When} | _] ->
+        [{{crashed, _Reason}, When} | _] = Crashes ->
+            BackoffExp = erlang:min(length(Crashes), ?MAX_BACKOFF_EXPONENT),
+            BackoffInterval = (1 bsl BackoffExp) * ?BACKOFF_INTERVAL_MICROS,
+            couch_log:notice("delay for ~p is now ~p s", [Job#job.id, BackoffInterval div 1000000]),
```

Comment:
remove log line, this'll just be noise. Perhaps move the logic here to a separate side-effect free function instead, so we can write some eunit tests to prove the progression of timeouts (and their cap).